tests/unit/states/test_archive.py

```diff
@@ -157,7 +166,7 @@ def test_tar_bsdtar(self):
         Tests the call of extraction with bsdtar
         '''
         bsdtar = MagicMock(return_value='tar (bsdtar)')
-        source = '/tmp/foo.tar.gz'
+        source = os.sep + os.path.join('tmp', 'foo.tar.gz')
```

Comment:
Same as the previous test. Does this one fail on Windows without `os.path.join()`?